junebug/command_line.py

```diff
@@ -61,6 +61,14 @@ def parse_arguments(args):
     parser.add_argument(
         '--amqp-password', '-amqppass', dest='amqp_pass', type=str,
         help='The password to use for the amqp auth. Defaults to "guest"')
+    parser.add_argument(
+        '--message-ttl', '-ttl', dest='ttl', type=int,
+        help='The maximum time allowed to reply to a message. Defaults to '
+        '"60" seconds')
```

Comment:
If we're going to be using the same ttl for both the inbound and outbound store, maybe the description should rather say something about the maximum time we store messages for. If we're using different ttls for each, maybe the option should be named to suggest it is for stored inbound messages. Similarly for the config field.